browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1156,11 +1159,11 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_RESET_BRAVE_REWARDS_ERROR_DESCRIPTION" desc="Description for Brave Rewards reset error dialog.">
         There was an error resetting Brave Rewards. Please relaunch Brave and try again.
       </message>
-      <message name="IDS_SHOW_ADS_WHEN_BRAVE_IS_NOT_IN_USE" desc="Title for Show Ads when Brave is not in use option in Brave rewards settings.">
-        Show Ads when Brave is not in use
+      <message name="IDS_SHOW_ADS_WHEN_BRAVE_IS_NOT_IN_USE_TITLE" desc="Title for Show Ads when Brave is not in use option in Brave rewards settings.">
+        Brave Notification Ads
```

Comment:
Nit: Is that the correct terminology now? The "notification" part seems superfluous/technical.